src/transformers/training_args.py

```diff
@@ -1065,6 +1066,17 @@ def main_process_first(self, local=True, desc="work"):
         else:
             yield
 
+    def get_warmup_steps(self, num_training_steps: int):
+        """
+        Get number of steps used for a linear warmup.
+        """
+        warmup_steps = (
+            self.warmup_steps
+            if self.warmup_steps > 0
+            else math.ceil(num_training_steps * self.warmup_ratio)
```

Comment:
Maybe just `int(num_training_steps * self.warmup_ratio)`

Reply:
Well, I copied this part from `trainer.py`.   If it is preferred to use `int` instead of `math.ceil`, we can remove `import math` on the top of `training_args.py`.

Reply:
Oh yeah, it's better to be consistent with `trainer.py`. Ignore what I just said!